Extension/src/LanguageServer/client.ts

```diff
@@ -2225,13 +2233,21 @@ export class DefaultClient implements Client {
         const message: string = notificationBody.status;
         util.setProgress(util.getProgressExecutableSuccess());
         const testHook: TestHook = getTestHook();
+        console.log("message: ", message);
         if (message.endsWith("Idle")) {
             // nothing to do
         } else if (message.endsWith("Parsing")) {
             this.model.isParsingWorkspace.Value = true;
+            this.model.isIndexingWorkspace.Value = message.includes("Discovering");
             this.model.isParsingWorkspacePausable.Value = false;
             const status: IntelliSenseStatus = { status: Status.TagParsingBegun };
             testHook.updateStatus(status);
+        } else if (message.endsWith("Initializing")) {
+            // Check which UI
+            if (ui instanceof NewUI) {
```

Comment:
I thought there was a property on the ui object that tells whether it's the new UI or not.